### PR TITLE
use the wrapped classes/apis in the seq store and fix when nothing there

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?189813

anyone

### CODE REVIEW

File: corehq/apps/hqadmin/utils.py [hunk 1]
```diff
@@ -8,9 +8,11 @@
 
 def pillow_seq_store():
     for pillow in get_all_pillow_instances():
-        checkpoint = pillow.get_checkpoint()
-        store, created = PillowCheckpointSeqStore.objects.get_or_create(checkpoint_id=checkpoint['_id'])
-        if not created and force_seq_int(checkpoint['seq']) < force_seq_int(store.seq) - EPSILON:
+        checkpoint = pillow.checkpoint
+        store, created = PillowCheckpointSeqStore.objects.get_or_create(checkpoint_id=checkpoint.checkpoint_id)
+        db_seq = checkpoint.get_current_sequence_id()
+        store_seq = force_seq_int(store.seq) or 0
```
Comment (czue) [inline corehq/apps/hqadmin/utils.py]: this was a separate issue that failed on the comparison if nothing was in the store.

**Comment by emord** [comment]
build fail looks related

```

======================================================================
ERROR: test_large_rewind (corehq.apps.hqadmin.tests.test_utils.TestPillowCheckpointSeqStore)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/hqadmin/tests/test_utils.py", line 61, in test_large_rewind
    pillow_seq_store()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/hqadmin/utils.py", line 21, in pillow_seq_store
    'pillow checkpoint seq': checkpoint['seq'],
TypeError: 'PillowCheckpoint' object has no attribute '__getitem__'
----------------------------------------------------------------------
Ran 251 tests in 1148.249s
```

**Comment by czue** [comment]
thanks, should be fixed (but good to wait)

**Comment by emord** [comment]
looks like same failure

**Comment by czue** [comment]
thanks, missed a file, is :white_check_mark: now

**Pull request merged**